tensorflow/core/grappler/optimizers/remapper.cc

```diff
@@ -149,6 +149,45 @@ struct ContractionWithBatchNormAndActivation {
   float epsilon = 0.0;
 };
 
+#ifdef INTEL_MKL
+// Contraction node followed by a BiasAdd and Add.
```

Comment:
1. Can you a disclaimer here, that though these patterns will be rewritten into the `_FusedConv2D` node, it does not really have a kernel implementation outside of MKL. That it's just a trick to trigger "real" rewrite rule in MKL layout pass.

2. Wouldn't it be easier to directly rewrite it into the MKL ops in remapper optimizer? My guess that MklLayoutPass takes care of required layout conversions.